near-sdk-macros/src/core_impl/abi/abi_generator.rs

```diff
@@ -41,6 +41,14 @@ impl ImplItemMethodInfo {
     /// If args are serialized with Borsh it will not include `#[derive(borsh::BorshSchema)]`.
     pub fn abi_struct(&self) -> TokenStream2 {
         let function_name_str = self.attr_signature_info.ident.to_string();
+        #[cfg(feature = "abi-doc")]
+        let function_doc =
+            match super::doc::parse_rustdoc(&self.attr_signature_info.non_bindgen_attrs) {
+                Some(doc) => quote! { Some(#doc.to_string()) },
+                None => quote! { None },
+            };
+        #[cfg(not(feature = "abi-doc"))]
+        let function_doc = quote! { None };
```

Comment:
I'm skeptical this should be a feature at all. This wouldn't be compiled to wasm32 so only really comes at the cost of trivial compile time for native targets. Feels a little unwieldy to enforce specific feature usage to enable rather than something that can be enabled/disabled when the ABI is generated (cargo-near). WDYT? Feels like there is a use case where someone might want to output docs _and_ not output docs for two different outputs for different use cases

Reply:
Indeed. Post-processing this makes more sense and even allows us to remove schemars generated doc strings. You can try this cargo-near branch in conjunction with this PR: https://github.com/near/cargo-near/tree/daniyar/rustdoc.